File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    float x;
    int n;
    int p;

while(true){

     printf("how nuch change is \n");
        x = get_float();
            if(x>0)
                {
                    break;
            }
        }
    n=x*100;
    p=0;
while(n>=25)
{
    p+=1;
    n=n-25;
}
while(n>=10)
{
    n=n-10;
      p+=1;
}
while(n>=5)
    {
        n=n-5;

      p+=1;
}
printf("%d\n",p+n);

}

File: pset6/cash/cash.py
from cs50 import get_float
from math import floor


def main():
		while True:
	    # getting dollars and cents
			dollars_owed = get_float("Change owed: ")
			cents_owed = floor(dollars_owed * 100)
	    # if the cent of ower equal to 0 stopped the program otherwise
			if cents_owed > 0:
			    	break
# divided each part
		quarters = cents_owed // 25
		# printing all divieded values
		dimes = (cents_owed % 25) // 10
		# printing all divieded values
		nickels = ((cents_owed % 25) % 10) // 5
		# printing all divieded values
		pennies = ((cents_owed % 25) % 10) % 5
# printing all divieded values
		print(f"{quarters + dimes + nickels + pennies}")


if __name__ == "__main__":
	main()